client/src/components/DatePicker/DatePicker.stories.js

```diff
@@ -5,12 +5,12 @@ import DatePicker from './DatePicker';
 
 function Child(props) {
     function handleChange(event) {
-        // Parent calls Child and props onChange
+        // Child component called by parent with props.onChange
         props.onChange(event.target.value);
     }
   
     return <input value={props.value} onChange={handleChange} />
 }
```

Comment:
What is this? Is this the onChange handler? Can you fix this so you can pass it into your `DatePicker` component and have it get called whenever the date picker changes?

Reply:
Modified with props.onChange and passed to DatePicker.js.